python/lsst/afw/typehandling/_Storable.cc

```diff
@@ -46,8 +46,9 @@ void wrapStorable(utils::python::WrapperCollection& wrappers) {
     wrappers.wrapType(PyStorable(wrappers.module, "Storable"), [](auto& mod, auto& cls) {
         // Do not wrap methods inherited from Persistable
         cls.def(py::init<>());  // Dummy constructor for pure-Python subclasses
-        cls.def("__copy__", &Storable::clone);
-        cls.def("__deepcopy__", [](Storable const& self, py::dict const& memo) { return self.clone(); },
+        cls.def("__copy__", &Storable::cloneStorable);
+        cls.def("__deepcopy__",
+                [](Storable const& self, py::dict const& memo) { return self.cloneStorable(); },
```

Comment:
Hmm, I guess I didn't explicitly specify that. To me, "copy" implies "deep copy". :confused: Though I do require that `cloneStorable` have the same behavior as `clone` if the latter exists, and I suppose there could be a shallow-copy `clone` in the Stack.

I consider `bool deep` an anti-pattern, given that:
* there's pretty much no reason to want a shallow copy if the class is mutable (or rather, you can't think of the result as a "copy"), whereas there's no reason to want a deep copy if the class is immutable
* a method that performs both shallow and deep copies, notwithstanding the above, is doing two different operations, violating the single responsibility principle
* default parameters are easy to overlook when learning an API by looking at the code; if the parameters control fundamental changes in what a function/method does, those changes may be hidden

I suppose we could remove these pybind11 bindings, since I don't think they're needed to use Python subclasses of `Storable` from C++ (though I'd rather wait until [DM-19582](https://jira.lsstcorp.org/browse/DM-19582) before swearing to that).